### ISSUE TITLE
npm install fails with node 5.6 on debian

### ISSUE THREAD

**Issue by diracdeltas** [brave/browser-laptop#734]
```
npm WARN optional Skipping failed optional dependency /electron-builder/appdmg:
npm WARN notsup Not compatible with your operating system or architecture: appdmg@0.3.5
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.7
npm ERR! Linux 4.3.0-1-amd64
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v5.6.0
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE

npm ERR! electron-prebuilt@0.36.7 postinstall: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the electron-prebuilt@0.36.7 postinstall script 'node install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the electron-prebuilt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs electron-prebuilt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls electron-prebuilt
npm ERR! There is likely additional logging output above.
```

Works fine again if I downgrade to 5.5

**Comment by bbondy** [comment]
I had to do the build yesterday (only on linux) on 5.5 btw.

**Comment by diracdeltas** [comment]
to clarify, by `build` i actually meant `npm install`.

**Comment by bbondy** [comment]
ditto

**Comment by weems** [comment]
I received the same error on OSX 10.11 + node 5.6 yesterday, too.

**Comment by luixxiul** [comment]
Fixed with #1682?

**Comment by luixxiul** [comment]
Should be fixed already, closing.
